yb-voyager/cmd/root.go

```diff
@@ -51,6 +58,14 @@ var rootCmd = &cobra.Command{
 			os.Exit(0)
 		}
 	},
+
+	PersistentPostRun: func(cmd *cobra.Command, args []string) {
+		if exportDir != "" && utils.FileOrFolderExists(exportDir) {
+			if cmd.Use != "version" {
+				unlockExportDir()
```

Comment:
Will this code be executed if voyager terminates with `os.Exit(...)` or `Ctrl+C`?

Reply:
I just kept the same code structure as the `PersistentPreRun` block here.

Reply:
I have added the changes you asked